lib/temporal.js

```diff
@@ -32,6 +32,10 @@ function Task(entry) {
   this.called = 0;
   this.now = this.calledAt = Date.now();
 
+  if (entry.id) {
+    this.id = entry.id;
+  }
```

Comment:
Let's give all `Task` instances an `id`! Keep an incrementing int in the outer scope, ie.

```js
var taskId = 1;

function Task(...) {
  ...
  this.id = entry.id || ++taskId;
  ...
}
```